Makefile

```diff
@@ -94,7 +94,7 @@ install-apis: generate ## Install the core rukpak CRDs
 install-plain: install-apis ## Install the rukpak CRDs and the plain provisioner
 	kubectl apply -f internal/provisioner/plain/manifests
 
-install: install-plain ## Install all rukpak core CRDs and provisioners
+install: cert-mgr install-plain ## Install all rukpak core CRDs and provisioners
```

Comment:
Sorry for missed in #156.  I think the install-api requires it.

Reply:
+1 to move this to the `install-apis` target. That's what actually depends on cert manager.

Reply:
@akihikokuroda Done. Thanks for pointing that out!